kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+  <div class="auth-message">
+    <h1>
+      {{ header || $tr('loginPrompt') }}
```

Comment:
This can be done for `header` but not for `details`. When you define a prop `default` as a function, you don't have access to any instance methods e.g. `defaultDetails`, but you do have access to `$tr`.